tla-types/src/test/scala/at/forsyte/apalache/tla/typecheck/etc/TestEtcTypeChecker.scala

```diff
@@ -899,6 +899,43 @@ class TestEtcTypeChecker extends AnyFunSuite with EasyMockSugar with BeforeAndAf
     }
   }
 
+  test("imprecise annotation") {
+    // Test case for: https://github.com/informalsystems/apalache/issues/2102
+
+    // let @type: (a, b) => Bool;
+    //   F(x, y) == x = y in TRUE
```

Comment:
This comment is a bit confusing to me because it is not valid TLA+, but nor is it exacly any other language I don't to think of....

Reply:
True. Let's make it TLA+